enhancements/console/customize-add-page.md

```diff
@@ -0,0 +1,171 @@
+
+A final yaml should look like this one:
+
+```yaml
+apiVersion: operator.openshift.io/v1
+kind: Console
+metadata:
+  name: cluster
+  ...
+spec:
+  customization:
+    brand: online
+    addPage:
+      hideActions:
+      - devcatalog/databases
+      - devcatalog/helmcharts
+      - devcatalog/operatorbacked
+      - devcatalog/virtualmachinetemplates
+      - devcatalog/managedservices
+      - fromgit
+      - fromdevfile
+      - fromdockerfile
+      - containerimage
+      - importyaml
+      - uploadajar
+      - knative.openshift.io/eventsource
+      - knative.openshift.io/channel
+      - tekton.dev/pipelines
+  ...
+```
+
+### Provide a code snippet in YAML editor
+
+The admin needs to know the current available actions to hide them. We don't want to provide the defaults in the CR so that we can change these defaults later without migrating the customer data.
+
+To provide the default values close to the editor, the console will provide a code snippets in the sidebar of the YAML editor when editing the `Console` operator config. The snippets should include the ids of all extensions.
+
+### User Stories
+
+#### Story 1
+
+As an cluster admin, I want to customize the add page. The admin needs to know the current list of available actions. The admin can configure a list of actions which should not be shown on the add page.
+
+#### Story 2
+
+As a user, I don't want to see any actions on the add page which are hidden by the cluster admin. Empty groups should not be displayed.
+
+### Risks and Mitigations
+
+#### How we handle upgrade
+
+**Mitigation**: The cluster admin can provide a list of actions which should not shown to the users. All features which are not defined in that list will be shown after an upgrade.
+
+**Drawback** If the cluster admin installs a new OpenShift upgrade the customized add page will contain new or unexpected actions.
+
+## Open Questions
+
+- [x] Should we save a list of enabled actions instead of a list of ids which should not be shown?
+
+For example:
+
+```yaml
+spec:
+  customization:
+    addPage:
+      actions:
+      - databases
+      - helmcharts
+      - ...
+  ...
+```
+
+Answer:
+
+> No. Customers requested to hide some unnecessary actions and add their own actions to the add page. Product management expected that it's easier for them to define a negative list, so that new actions will be shown automatically if an operator is installed.
+
+- [x] Should we also allow the cluster admin to hide complete groups?
+
+Answer:
+
+> No. Groups are shown based on whether an action is present or not. It's fine that the admin needs to disable new actions after a cluster upgrade.
+
+- [x] Should we save which actions are already configured by the cluster admin?
+
+If we save a list of shown and hidden actions, its maybe easier later to show an admin the information which new actions are available after an upgrade or when a new operator is installed.
```

Comment:
```suggestion
If we save a list of shown and hidden actions, it may be easier later to show an admin the information which new actions are available after an upgrade or when a new operator is installed.
```

Reply:
Thanks again :)